symbiflow-xc7a50t/meta.yaml

```diff
@@ -0,0 +1,21 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: symbiflow-xc7a50t
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SymbiFlow/conda-packages.git
```

Comment:
I think this URL should probably be symbiflow-arch-defs?

Reply:
Sure, the only issue is that the version reported in the package (commit has and so) will be different from the tarball version unless we can have the tarball placed in a `latest` directory in GCS, so that each time the arch-defs HEAD changes, also the tarball does and we get the latest version of the tool.

Currently, there is a huge path to access the tarball from GCS (e.g. https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/206/20200526-034850/symbiflow-arch-defs-install-97519a47.tar.xz).

Reply:
The contents should match the version number?